### DISCUSSION TITLE
Delight: What is a homeserver? FAQ page

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#526] [Ideas]
### Your use case

https://www.figma.com/file/ZDVvKC8odsa5Jeml9gCuLd/Web%3A-New-User's-Journey?node-id=200%3A2780

**Comment by germain-gg** [comment]
We will need @JordanConnor help to embed this on the website.

@daniellekirkwood Who would be in charge of coming up with the copy for this new page?

**Reply by JordanConnor** [reply]
We'd be adding this to element.io FAQ? Happy to help when the copy is ready :)

**Comment by daniellekirkwood** [comment]
I'm changing the tags from Needs-Design to Needs-Info and Needs-Product as I'll work on updating this issue so the problem and solution is much clearer

**Comment by t3chguy** [comment]
If this is an issue for the website can we please move it to https://gitlab.matrix.org/new-vector/element/element.io/-/issues?sort=created_date&state=opened - it'd be great to keep the hygiene of this repo about the Element Web codebase

**Comment by daniellekirkwood** [comment]
I'll move it to Meta. It applies to all platforms and is currently a PM and Design task

**Comment by germain-gg** [comment]
@thibaultamartin I remember you talking about an onboarding document for new users on Element. This is one of the task that we identified as really odd in our registration flow.

Is an "what is a homeserver" page on your list? If so, we can link to it once it's live

**Comment by thibaultamartin** [comment]
Yes I'll keep you posted when it's live! FWIW, given the documentation is meant for non-technical people, I introduce the concept of homeserver by calling them (Matrix/service) providers

**Comment by daniellekirkwood** [comment]
This is a great idea and we need to offer more guidance on the technical aspects of our product but as we're not actively working on this right now, I'm going to transfer it to "Discussions"
